quotas.py

```diff
@@ -104,7 +104,9 @@ def _get_single_dict(nova, neutron, cinder):
         nova = nova.to_dict()
     
     if type(cinder) is not dict:
-        cinder = cinder.to_dict()
+        # cinder client lacks to_dict() function. fixed in upstream mitaka
+        # but not in redhot mitaka
```

Comment:
Oops, this should say "redhat" or "Red Hat" or not "redhot".  Although redhot would be a great name for a piece of software. 

Maybe capitalize Mitaka while you are making a change anyway.